content/docs/setup/kubernetes/quick-start-alicloud-ack/index.md

```diff
@@ -5,49 +5,57 @@ weight: 12
 keywords: [kubernetes,alicloud,aliyun]
 ---
 
-Quick Start instructions to install and run Istio in [AliCloud Kubernetes Container Service](https://cs.console.aliyun.com/) using Application Catalog.
+Follow these instructions to install and run Istio in the 
+[AliCloud Kubernetes Container Service](https://cs.console.aliyun.com/) 
+using the `Application Catalog`.
 
-This Quick Start installs the current release version of Istio and then deploys the [Bookinfo](/docs/examples/bookinfo/) sample
-application.  It uses Application Catalog to automate the steps detailed in the [Istio on Kubernetes setup guide](/docs/setup/kubernetes/quick-start/) for Kubernetes Engine.
+This guide installs the current release version of Istio and deploys the 
+[Bookinfo](/docs/examples/bookinfo/) sample application.  
+To automate the steps detailed in the [Istio on Kubernetes setup guide](/docs/setup/kubernetes/quick-start/)
+for Kubernetes Engine.
 
 ## Prerequisites
 
-- This sample assumes that you already have an avaiable AliCloud Kubernetes cluster. Otherwise, you can create a Kubernetes cluster quickly and easily in the Container Service console.
+- You have an avaiable AliCloud Kubernetes cluster. Otherwise, create a 
+Kubernetes cluster quickly and easily in the `Container Service console`.
 
-- Make sure that `kubectl` can work fine for your Kubernetes cluster.
+- `kubectl` works fine for your Kubernetes cluster.
 
-- Create a namespace to deploy Istio components, e.g. `istio-system`:
+- You can create a namespace to deploy Istio components. The following example
+ creates the `istio-system` namespace:
 
 {{< text bash >}}
     $ kubectl create namespace istio-system
```

Comment:
Make sure the commands have the same indentation as the {{< text bash >}} markup. Otherwise, the linter will complain.

Reply:
see the new commit 4ab4781